wildlifelicensing/apps/applications/templates/wl/conditions/enter_conditions.html

```diff
@@ -74,7 +74,23 @@ <h4 class="inline">
                                     <p>
                                         {{ applicant.user.first_name }} {{ applicant.user.last_name }}
                                     </p>
-                                    <p>{{ applicant.postal_address.search_text }}</p>
+                                    <div>{{ applicant.postal_address.search_text }}</div>
+                                    <div class="applicant-contact">
+                                        <div>
+                                            <span class="fa fa-envelope"></span>
+                                            <span class="email"> {{ applicant.email }}</span>
+                                        </div>
+                                        {% if applicant.user.phone_number %}
+                                            <div>
+                                                <span class="fa fa-phone phone"></span> {{ applicant.user.phone_number }}
+                                            </div>
+                                        {% endif %}
+                                        {% if applicant.user.mobile_number %}
+                                            <div>
+                                                <span class="fa fa-mobile mobile"></span> {{ applicant.user.mobile_number }}
+                                            </div>
+                                        {% endif %}
+                                    </div>
```

Comment:
I know, DRY! This should be put in a shared included template.
I will do it if you ask. :-)